modules/client/examples/04-decoders/15-add-multisig-addresses.ts

```diff
@@ -0,0 +1,28 @@
+/* MARKDOWN
+### Decode Add Members Action (Multisig)
+*/
+import { Context, ContextPlugin, MultisigClient } from "@aragon/sdk-client";
+import { MultisigPluginSettings } from "../../src";
+import { contextParams } from "../00-client/00-context";
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+const multisigClient = new MultisigClient(contextPlugin);
+const data: Uint8Array = new Uint8Array([12, 56]);
+
+const settings: MultisigPluginSettings = multisigClient.decoding
+  .addAddressesAction(
+    data,
+  );
+
+console.log(settings);
+/*
+{
+  members: [
+    "0x12345...",
+    "0x56789...",
+    "0x13579...",
+  ],
+  minApprovals: 2n,
```

Comment:
How do we get members + minApprovals when the encoding action only specifies members?

We should be encoding both with the new minApprovals amount

Reply:
It follows the naming pattern in the contracts, when you add or remove members you should also specify the new minimum participation. 

Maybe leaving the participation as a percentage in the contracts would make more sense to avoid the need of changing the min approvals each time we add or remove an address